### PR TITLE
CVE & CPE dictonary update via URL

### PR DESCRIPTION
**Pull request by b3n4kh**
This PR adds the option to download cve and cpe xml via an URL, and fixes various bugs that were related to that function.

* monkeypatch remaining services as in https://github.com/SK-CERT/Taranis-NG/pull/65 necesarry since python requests triggers the issue
* Add new environment variables `CVE_UPDATE_URL` and `CPE_UPDATE_URL` allowing automatic download of dictionaries
* Fix sqlalchemy bug during import https://sqlalche.me/e/14/qzyx
* Fix typo debug_log() => log_debug() globally
* Fix logging in manager.py

### CODE REVIEW

File: src/core/manage.py [hunk 1]
```diff
@@ -355,6 +365,26 @@ def upload_to(self, filename):
             app.logger.critical("Upload failed!")
             abort()
 
+    def download_update_file(self, filename, url):
+        import requests
+        import gzip
```
Comment (sebix) [inline src/core/manage.py:370]: Why lazy importing?

File: src/core/manage.py [hunk 2]
```diff
@@ -355,6 +365,26 @@ def upload_to(self, filename):
...
+          r = requests.get(url, allow_redirects=True)
+          content_type = r.headers.get('Content-Type')
+          content_enc = r.headers.get('Content-Encoding')
+          if content_type == 'application/x-gzip' or content_enc == 'x-gzip':
+            content = gzip.decompress(r.content)
+          else:
+            if content_type != 'application/xml':
+              app.logger.debug("{0} does not point to an XML content-typed file".format(url))
+            content = r.content
+          open(filename, 'wb').write(content)
```
Comment (sebix) [inline src/core/manage.py:383]: Missing context
Is writing as bytes necessary?

Comment (b3n4kh) [reply src/core/manage.py:383]: decission should match the "old" behaviour which uses 'wb'
Could be reevaluated to swich both

File: src/core/manage.py (hunk 1 above)
Comment (b3n4kh) [reply src/core/manage.py:370]: whole file uses lazy importing quit a lot
maybe refactor during https://github.com/SK-CERT/Taranis-NG/issues/71

**Pull request closed without merging**